### DISCUSSION TITLE
Update your theme without losing customisation (from 2.1 to latest)

### DISCUSSION THREAD

**Discussion by sparh** [Oksydan/falcon#374] [Q&A]
Hello
I have your theme 2.1.0 installed and working very well on a PS 1.7.8.10.
I would like to update my PS version to the latest 8.1 so I need to update also your theme [https://github.com/Oksydan/falcon?tab=readme-ov-file#support-table](https://github.com/Oksydan/falcon?tab=readme-ov-file#support-table)

Since I have made many customisations to your theme (tpl, css and js) do you have some kind of procedure on how to update it without losing all the customisation I have made.

Regards

**Comment by Oksydan** [comment]
Hi @sparh,

the best way but a manual way is to compare two releases https://github.com/Oksydan/falcon/compare/v2.1.0...v3.1.2
You can check and apply changes by files or commits. I know that this might not be the 'easiest' solution but there is not other way to do it 😕. Also remember to upgrade Falcon modules to the latest version.

**Comment by sparh** [comment]
Hello 
thanks for this.

Should I update first Prestashop and then your theme and modules or the other way (first your theme and modules and then Prestashop)

Regards

**Reply by Oksydan** [reply]
Hi, 
🤔 I think you should upgrade your PrestaShop app first. Then go for module and theme. You have to take a lot at the modules. There might be some changes in template so adjustments might be required especially if you overridden them in theme directory.
